test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -433,7 +433,7 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 
 		//fetch the latest profile
 		profile, err := profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
-		Expect(err).ToNot(HaveOccurred())
+		//Expect(err).ToNot(HaveOccurred())
```

Comment:
if this line is not needed then remove it , although err should be checked.

Reply:
it's not clear to me why this line is causing an issue.